### PR TITLE
s390x: Adapt zfcp test to test both adapters

### PR DESCRIPTION
**Pull request by Soulofdestiny**
[related poo](https://progress.opensuse.org/issues/30550)
[related needles](https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/734)
[verification run](http://opeth/tests/148)

### CODE REVIEW

File: tests/installation/disk_activation.pm [hunk 1]
```diff
@@ -23,24 +23,35 @@ sub format_dasd {
...
+    assert_screen 'zfcp-channel-select';
+    for (1 .. 8) { send_key "backspace"; }
+    type_string "$channel";
+    assert_screen "zfcp-channel-$channel-selected";
+
+    send_key $cmd{next};
+
+    # use allow_lun_scan
+    # popup was removed for sle15, allow_lun_scan still works though
+    if (!is_sle '15+') {
```
Comment (rwx788) [inline tests/installation/disk_activation.pm]: Now we can write is_sle('<15') to do these checks

File: tests/installation/disk_activation.pm [hunk 2]
```diff
@@ -23,24 +23,35 @@ sub format_dasd {
...
 }
 
+sub add_zfcp_disk {
+    my $channel = shift;
+    assert_screen 'zfcp-disk-management';
+    send_key 'alt-a';
+    assert_screen 'zfcp-add-device';
+    send_key 'alt-a';
+    assert_screen 'zfcp-channel-select';
+    for (1 .. 8) { send_key "backspace"; }
```
Comment (rwx788) [inline tests/installation/disk_activation.pm]: What's about pressing ctrl-a + del to clean the field? I would also wrap it in wait_screen_change, so we don't start typing before removed the content.

File: tests/installation/disk_activation.pm [hunk 3]
```diff
@@ -110,7 +121,7 @@ sub run {
     send_key $cmd{next};
 
     # check for multipath popup
-    if (check_screen('detected-multipath', 5)) {
+    if (check_screen('detected-multipath', 10)) {
```
Comment (okurz) [inline tests/installation/disk_activation.pm]: was ugly, is still ugly

**Comment by okurz** [comment]
needles merged

File: tests/installation/disk_activation.pm (hunk 1 above)
Comment (Soulofdestiny) [reply tests/installation/disk_activation.pm]: changed

File: tests/installation/disk_activation.pm (hunk 2 above)
Comment (Soulofdestiny) [reply tests/installation/disk_activation.pm]: from experience on z/VM key combinations with key modifiers might be not 100% reliable...
I'd go with this one for now

**Comment by Soulofdestiny** [comment]
updated needles for sp4 (only one differs actually)

File: tests/installation/disk_activation.pm (hunk 2 above)
Comment (okurz) [reply tests/installation/disk_activation.pm]: Should we not try to understand what key presses go missing over VNC?

**Pull request merged**